src/Components/DisplayBooks.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Store from '../Store/Store';
+import Books from './Shared/Books';
+
+const DisplayBooks = () => Store.map((book) => (
+  <Books key={book.id} title={book.title} author={book.author} />
+));
+
+export default DisplayBooks;
```

Comment:
- [ ] This component should contain the functionality for listing all books and also displaying the AddBook component. Kindly remove the the `AddBook` component from the App component and add it here instead:

  Suggested code.
  ```
  import React from 'react';
  import Store from '../Store/Store';
  import Books from './Shared/Books';
  import AddBook from './AddBook';
  
  const DisplayBooks = () => (
    Store.map((book) => (
      <>
        <Books key={book.id} title={book.title} author={book.author} />
        <AddBook />
      </>
    ))
  );
  
  export default DisplayBooks;
  ```